packages/examples/src/bare/client.ts

```diff
@@ -15,24 +15,24 @@ import { WebSocketMessageReader, WebSocketMessageWriter, toSocket } from 'vscode
 import { CloseAction, ErrorAction, MessageTransports } from 'vscode-languageclient/browser.js';
 import { configureMonacoWorkers } from '../common/client/utils.js';
 import { ConsoleLogger } from 'monaco-languageclient/tools';
-import { updateUserConfiguration } from '@codingame/monaco-vscode-configuration-service-override';
 
 export const runClient = async () => {
     const logger = new ConsoleLogger(LogLevel.Debug);
     const htmlContainer = document.getElementById('monaco-editor-root')!;
     await initServices({
         serviceOverrides: {
             ...getConfigurationServiceOverride()
-        }
+        },
+        userConfiguration: {
```

Comment:
what about renaming it to `defaultUserConfiguration`?

Reply:
It is an API change and basically everything you pass is the default. I would keep it like this for now.